.github/workflows/int-and-connected-test-run.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           base64 --decode <<< ${{ secrets.SA_B64_CREDENTIALS }} > ${GOOGLE_APPLICATION_CREDENTIALS}
           base64 --decode <<< ${{ secrets.B64_RBS_APPLICATION_CREDENTIALS }} > ${RBS_CLIENT_CREDENTIAL_FILE_PATH}
           # assemble code and run connected tests
-          ./gradlew --scan assemble testConnected
```

Comment:
I don't think that order matters.  I moved scan to the end mostly because it matches what we have in the datarepo-actions repo and I didn't want to have to iterate on it given the long test run time, but it looks like it matches what is in the [gradle docs](https://docs.gradle.org/current/userguide/build_scans.html#enable_build_scans). 
And, I think Phil is right, we can remove assemble. I'm going to leave so that I don't have to wait for another test run, but I'll remove it with my connected test work.